tests/sentry/hybrid_cloud/test_organizationmapping.py

```diff
@@ -82,11 +83,72 @@ def test_update_customer_id(self):
         api_org_mapping = organization_mapping_service.create(**fields)
         assert api_org_mapping.customer_id is None
 
-        assert (
-            organization_mapping_service.update_customer_id(
-                organization_id=self.organization.id, customer_id="test"
+        organization_mapping_service.update(
+            ApiOrganizationMappingUpdate(
+                organization_id=self.organization.id,
```

Comment:
the UnsetVal type would be nice for initializers like this

Reply:
https://github.com/getsentry/sentry/pull/42687/commits/ed9eec578d5ebd4b829e146ae458c6d8e710279d